### PR TITLE
libsass implementation and sass and foundation updates

### PR DESCRIPTION
**Pull request by anilop**
RARO-41 Before we merge this i want someone to try and build this branch on linux and mac.

### CODE REVIEW

File: src/main/webapp/WEB-INF/themes/desktop/gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+'use strict';
```
Comment (sjtower) [inline src/main/webapp/WEB-INF/themes/desktop/gulpfile.js]: Please add a note if this is a libsass-only file and I don't have to care about it.

File: src/main/webapp/WEB-INF/themes/desktop/package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (sjtower) [inline src/main/webapp/WEB-INF/themes/desktop/package.json]: Same here - please add a comment note

File: src/main/webapp/WEB-INF/themes/desktop/sass/_utilities.scss [hunk 3]
```diff
@@ -379,4 +379,50 @@ example | extension
...
+
+@function image-url($src,$prefixImagePath:true,$cacheBuster:false){
+
+  @if(type-of($src) == string){
+       $url:false; // must define
+    $src: #{$src};
+    @if($prefixImagePath == true){
+          $url: '#{$image-path}#{$src}';
+       }
+               @else
```
Comment (sjtower) [inline src/main/webapp/WEB-INF/themes/desktop/sass/_utilities.scss]: Please fix spacing/indents

File: src/main/webapp/WEB-INF/themes/desktop/sass/_utilities.scss [hunk 4]
```diff
@@ -379,4 +379,50 @@ example | extension
...
+       }
+        @if($cacheBuster == true){
+           $url: '#{$url}#{$cache-buster}';
+       }
+        @return url($url);
+    }
+}
+
+
+@function font-files($src,$prefixImagePath:true,$cacheBuster:false){
```
Comment (sjtower) [inline src/main/webapp/WEB-INF/themes/desktop/sass/_utilities.scss]: Please add documentation for these methods

File: src/main/webapp/WEB-INF/themes/desktop/sass/sections/_header.scss [hunk 5]
```diff
@@ -152,6 +157,8 @@ $width-nav: $block-grid-elements - $width-logo;
   }
 
 .top-bar-section {
+
+
```
Comment (sjtower) [inline src/main/webapp/WEB-INF/themes/desktop/sass/sections/_header.scss]: Are these various newlines intentional?

File: src/main/webapp/WEB-INF/themes/desktop/sass/vendor/compass/animation/animate/bouncing/_bouncing-entrances.scss [hunk 6]
```diff
@@ -0,0 +1,68 @@
+// ---------------------------------------------------------------------------
```
Comment (sjtower) [inline src/main/webapp/WEB-INF/themes/desktop/sass/vendor/compass/animation/animate/bouncing/_bouncing-entrances.scss]: Do we really need these "bouncing" animation scss files? I doubt we'll use them anytime soon.

Comment (anilop) [inline src/main/webapp/WEB-INF/themes/desktop/sass/vendor/compass/animation/animate/bouncing/_bouncing-entrances.scss]: We do need them, they are a library and not compiled.

File: src/main/webapp/WEB-INF/themes/desktop/sass/_utilities.scss (hunk 4 above)
Comment (anilop) [inline src/main/webapp/WEB-INF/themes/desktop/sass/_utilities.scss]: done

File: src/main/webapp/WEB-INF/themes/desktop/package.json (hunk 2 above)
Comment (anilop) [inline src/main/webapp/WEB-INF/themes/desktop/package.json]: Comments are not allowed in JSON files

**Pull request merged**